pkgs/development/python-modules/abjad/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ply
+, roman
+, uqbar
+}:
+
+buildPythonPackage rec {
+  pname = "abjad";
+  version = "3.19";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-I9t3ORUKFFlMfXJsAzXhCzl1B4a9/ZNmvSX2/R44TPs=";
+  };
+
+  propagatedBuildInputs = [
+    ply
+    roman
+    uqbar
+  ];
+
```

Comment:
Could you please enable tests?
We need to add `pytestCheckHook` to `nativeCheckInputs`.